main.go

```diff
@@ -65,16 +65,36 @@ func main() {
 			Usage:  "AWS ECS cluster",
 			EnvVar: "PLUGIN_CLUSTER",
 		},
+		cli.StringFlag{
+			Name:   "log-driver",
+			Usage:  "The log driver to use for the container",
+			EnvVar: "PLUGIN_LOG_DRIVER",
+		},
+		cli.StringSliceFlag{
+			Name:   "log-options",
+			Usage:  "The configuration options to send to the log driver",
+			EnvVar: "PLUGIN_LOG_OPTIONS",
+		},
 		cli.StringSliceFlag{
 			Name:   "port-mappings",
 			Usage:  "ECS port maps",
 			EnvVar: "PLUGIN_PORT_MAPPINGS",
 		},
 		cli.StringSliceFlag{
+			Name:   "docker-labels",
+			Usage:  "A key/value map of labels to add to the container",
+			EnvVar: "PLUGIN_DOCKER_LABELS",
+		},
+		cli.StringSliceFlag{
 			Name:   "environment-variables",
 			Usage:  "ECS environment-variables",
 			EnvVar: "PLUGIN_ENVIRONMENT_VARIABLES",
 		},
+		cli.StringSliceFlag{
+			Name:   "secret-environment-variables",
+			Usage:  "Secret ECS environment-variables",
+			EnvVar: "PLUGIN_SECRET_ENVIRONMENT_VARIABLES",
+		},
```

Comment:
Should probably also have ECS_SECRET_ENVIRONMENT_VARIABLES so that it can be injected as a secret and not through the yml file